files/etc/profile.d/workarea.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+#
+# Configure all login shells to use the settings in `/etc/workarea.env`.
+# Set up the Workarea application's configuration if this is the first
+# time someone has logged in.
+#
+
+set -a
+. /etc/workarea.env
+set +a
+
+setup() {
+  local ip
+  local host
+  local access_key
+  local secret_key
+  local bucket_name
+  local password
+
+  echo
+  echo "Welcome to your new Workarea application!"
+  echo
+  echo "There are just a few options you need to set..."
+  echo
+
+  echo -n "Enter your AWS S3 Access Key: "
+  read -r access_key
+  echo -n "Enter your AWS S3 Secret Key: "
+  read -r secret_key
+  echo -n "Enter your AWS S3 Bucket Name: "
+  read -r bucket_name
+
+  ip="$(curl -s https://ipinfo.io/ip)"
+  echo -n "Enter your Hostname (default $ip): "
+  read -r host
+  echo
+  if [[ -z $host ]]; then
+    host=$ip
+  fi
+
+  echo "Applying configuration..."
+  {
+    echo "WORKAREA_S3_ACCESS_KEY_ID=$access_key"
+    echo "WORKAREA_S3_SECRET_ACCESS_KEY=$secret_key"
+    echo "WORKAREA_S3_BUCKET_NAME=$bucket_name"
+    echo "WORKAREA_HOST=$host"
+  } >> /etc/workarea.env
+  set -a
+  . /etc/workarea.env
+  set +a
+  sed -i "s/www.shop.com/$host/" /srv/shop/config/initializers/workarea.rb
```

Comment:
this fixes the issue where you generate the app in a folder called `shop` and it redirects you to `www.shop.com` when you boot the app :)